exist-core/src/test/java/org/exist/util/ConfigurationTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * eXist-db Open Source Native XML Database
+ * Copyright (C) 2001 The eXist-db Authors
+ *
+ * info@exist-db.org
+ * http://www.exist-db.org
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.exist.util;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+
+class ConfigurationTest {
+    @TempDir
+    Path existHomeDir;
+
+    @Test
+    void testConfigurationConstructors() throws Exception {
+        assertThatNoException().isThrownBy(Configuration::new);
+        assertThatNoException().isThrownBy(() -> new Configuration(null));
+        assertThatNoException().isThrownBy(() -> new Configuration(writeConfig(existHomeDir.resolve("myconf.xml")), Optional.of(existHomeDir)));
+    }
+
+    private String writeConfig(Path confFile) throws IOException {
```

Comment:
I think it would be better to add this as a file to the test resources folder, name it appropriately and load it at test execution time. There should be some existing examples in the tests of how to do this...